migrations/20250227_viz_move/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#!/bin/bash
+set -eu
+CONTAINER_NAME="beebop-api"
+BACKUP_FOLDER=/beebop/storage/poppunk_output_backup
+
+# Remove backup_folder inside the container if exists
+if docker exec $CONTAINER_NAME test -d $BACKUP_FOLDER; then
+    docker exec $CONTAINER_NAME rm -rf $BACKUP_FOLDER
+    echo "Successfully restored PopPUNK output from backup."
+else
+    echo "Backup folder $BACKUP_FOLDER does not exist inside the container."
+fi
+
+# Remove migration.py inside the container
+docker exec $CONTAINER_NAME rm -f /beebop/storage/migration.py
```

Comment:
If this is the point of no return for the migration, perhaps this is also where you should be deleting the old redis keys too? I don't think that's happening otherwise..?

Reply:
oh yup good idea